JS.md

```diff
@@ -29,6 +29,21 @@ Because JavaScript exceptions can be handled, and JavaScript can continue to
 call WebAssembly exports after a trap has been handled, traps do not, in
 general, prevent future execution.
 
+Even though the `stack` property of JS `Error` objects is not specified by
+ES-262, implementations *should* include WebAssembly stack frames in
+the generated callstacks. [User-defined sections](BinaryEncoding.md#high-level-structure)
+may arbitrarily affect the displayed stack frames. However, by default, when
+no relevant sections are present:
+* The line number reported for a WebAssembly frame *should* be the offset of the 
+  the trap/call instruction within the module's originally-compiled binary.
+* The column number reported for WebAssembly frames, if not omitted entirely,
+  *should* be 1.
+* The function name *should* be generated from the template string literal 
+  `wasm-function[${i}]` where `i` is the index of the function in the module's
+  [Code section](BinaryEncoding.md#code-section).
+* The filename *should* be that of the JS caller of the originating `Module`
+  constructor call.
```

Comment:
The only JS APIs we have atm take an `ArrayBuffer` of bytes so I think this is the best we can do.  With ES6 module integration, we could have the same URLs as JS &lt;script&gt;s.  With Streams integration, if you `WebAssembly.compile(stream)`, maybe we could use the URL that was fetched to produce `stream`.